### ISSUE TITLE
ERROR in The target entry-point "@angular/material/card" has missing dependencies:  - @angular/animations  - @angular/animations/browserbug(COMPONENT): TITLE

### ISSUE THREAD

**Issue by priyadarsh1** [angular/components#20423]
#### Reproduction

Use StackBlitz to reproduce your issue: https://stackblitz.com/fork/components-issue

Steps to reproduce:
1. create a new project
2. npm install --save @angular/material @angular/cdk @angular/animations
 
 
#### Expected Behavior

What behavior were you expecting to see?
I expected my project to run fine

#### Actual Behavior

What behavior did you actually see?
ERROR in ./src/styles.css (./node_modules/css-loader/dist/cjs.js??ref--13-1!./node_modules/postcss-loader/src??embedded!./src/styles.css)
Module build failed (from ./node_modules/postcss-loader/src/index.js):
Error: Failed to find 'bootstrap/dist/css/bootstrap.css'
  in [
    C:\Users\hp\Desktop\autoPod\frontend\src
  ]
    at C:\Users\hp\Desktop\autoPod\frontend\node_modules\postcss-import\lib\resolve-id.js:35:13
    at async Promise.all (index 0)

ERROR in multi ./src/styles.css ../node_modules/bootstrap/dist/css/bootstrap.css
Module not found: Error: Can't resolve 'C:\Users\hp\Desktop\autoPod\node_modules\bootstrap\dist\css\bootstrap.css' in 'C:\Users\hp\Desktop\autoPod\frontend'

ERROR in The target entry-point "@angular/material/card" has missing dependencies:
 - @angular/animations
 - @angular/animations/browser

#### Environment

- Angular: 9.0.7
- CDK/Material: 9.2.4
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu):  Windows

**Comment by jdell64** [comment]
I have a similar issue, can't reproduce on stackblitz:

https://stackoverflow.com/questions/63599070/the-target-entry-point-angular-cdk-platform-has-missing-dependencies

```
ERROR in The target entry-point "@angular/cdk/platform" has missing dependencies:

@angular/core
@angular/common
```

Mine is version 10 though.

**Comment by devversion** [comment]
@jdell64 Would it be possible to have a standalone reproduction for this? I don't think a StackBlitz would be necessary. I realize that ngcc issues might not be reproducible in StackBlitz.

**Comment by devversion** [comment]
We run tests for ngcc compatibility of all our release packages, and also rely on ngcc for our docs site. That's why it seems unexpected to see this error. It might be also helpful if you share the contents of the tsconfig you pass to `ngcc`. The tsconfig is used by ngcc to determine the entry-points and dependencies.

**Comment by jdell64** [comment]
I tried... I can't seem to replicate it, and the app is proprietary :(
I restarted my update (from v7). v8 worked fine. This problem appeared in v9. I tried copying over a tsconfig file from a brand new v9 project, and still no luck.

**Comment by jdell64** [comment]
Okay, mine fails when I re-ran the migrations:
Undecorated classes with DI migration.

```
  As of Angular 9, it is no longer supported to use Angular DI on a class that does not have an Angular decorator.
  Read more about this here: https://v9.angular.io/guide/migration-undecorated-classes
Failed to read JSON file /myDevFolder/dev/node_modules/@angular/material/typings/core/index.metadata.json
    
    This migration uses the Angular compiler internally and therefore projects that no longer build successfully after the update cannot run the migration. Please ensure there are no AOT compilation errors and rerun the migration. The following project failed: src/tsconfig.app.json
    
    Error: Path "/../node_modules/@angular/material/typings/core/index.metadata.json" is invalid.
    
    Could not migrate all undecorated classes that use dependency
    injection. Some project targets could not be analyzed due to
    TypeScript program failures.
```

Could this be the cause? I ultimately want to get to v10, but I figured it would be better not to skip ahead.

**Comment by jdell64** [comment]
I wish I could have found the issue, but we ended up just copying our app folder to a new angular project

**Comment by devversion** [comment]
> I tried... I can't seem to replicate it, and the app is proprietary :(
I restarted my update (from v7). v8 worked fine. This problem appeared in v9. I tried copying over a tsconfig file from a brand new v9 project, and still no luck.

Oh, that's unfortunate. No worries. Thanks for trying!

> Okay, mine fails when I re-ran the migrations:
Undecorated classes with DI migration.

It's an interesting error you shared. I'm not quite sure why that happens. Unless the path here to the `@angular/material` package is incorrect, it sounds like a more recent version of Angular Material is installed. We removed the `typings` folder as of V9.

> I wish I could have found the issue, but we ended up just copying our app folder to a new angular project

Sorry to hear that. Thanks a lot for the information you shared. I'm going to close this issue as I haven't seen any related issue for this and wasn't able to reproduce this. The migrations also moved along, and more evolved versions should be in V9/V10.
